frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx

```diff
@@ -0,0 +1,463 @@
+        <Row>
+          <Text
+            style={{
+              margin: "6px 12px",
+            }}
+            type="warning"
+          >
+            {messages["annotation.python_do_not_share"]}
+          </Text>
+        </Row>
+      );
+    }
+    return null;
+  };
+
+  const handleTabChange = (key: string) => {
+    setActiveTabKey(key);
+  };
+
+  const handleCheckboxChange = (checkedValues: CheckboxValueType[]) => {
+    setIncludeVolumeData(checkedValues.includes("Volume"));
+  };
+
+  const handleKeepWindowOpenChecked = (e: any) => {
+    setKeepWindowOpen(e.target.checked);
+  };
+
+  const workerInfo = (
+    <Row>
+      <Divider
+        style={{
+          margin: "18px 0",
+        }}
+      />
+      <Text
+        style={{
+          margin: "6px 12px",
+        }}
+        type="warning"
+      >
+        {messages["annotation.export_no_worker"]}
+        <a href="mailto:hello@webknossos.com">hello@webknossos.com.</a>
+      </Text>
+    </Row>
+  );
+
+  const checkboxStyle = {
+    height: "30px",
+    lineHeight: "30px",
+  };
+
+  const authToken = useFetch(getAuthToken, "loading...", []);
+  const wkInitSnippet = `import webknossos as wk
+
+with wk.webknossos_context(token="${authToken}"):
+    annotation = wk.Annotation.download(
+        "${annotationId}",
+        annotation_type="${annotationType}",
+        webknossos_url="${window.location.origin}"
+    )
+`;
+
+  const alertTokenIsPrivate = () => {
+    Toast.warning(
+      "The clipboard contains private data. Do not share this information with anyone you do not trust!",
+    );
+  };
+
+  const selection = ["Volume", "Skeleton"];
+
+  return (
+    <Modal
+      title="Download this Annotation"
+      visible={isVisible}
+      width={600}
+      footer={[<Footer tabKey={activeTabKey} onClick={handleOk} key="footer" />]}
+      onCancel={onClose}
+      style={{ overflow: "visible" }}
+    >
+      <Tabs activeKey={activeTabKey} onChange={handleTabChange} type="card">
+        <TabPane tab="Download" key="download">
+          <Row>
+            {maybeShowWarning()}
+            <Text
+              style={{
+                margin: "6px 12px",
+              }}
+            >
+              {messages["annotation.download"]}
+            </Text>
+          </Row>
+          <Divider
+            style={{
+              margin: "18px 0",
+            }}
+          >
+            Options
+          </Divider>
+          <Row>
+            <Col
+              span={9}
+              style={{
+                lineHeight: "20px",
+                padding: "5px 12px",
+              }}
+            >
+              Select the data you would like to download.
+            </Col>
+            <Col span={15}>
+              <CheckboxGroup onChange={handleCheckboxChange} defaultValue={selection}>
+                <Checkbox style={checkboxStyle} value="Volume" checked={includeVolumeData}>
+                  Volume Annotations as WKW
+                </Checkbox>
```

Comment:
This checkbox shouldn't be there in case of skeleton-only annotations (you can use `hasVolumeTracings(state.tracing)` for that).